.gitignore

```diff
@@ -1,4 +1,5 @@
-/.deps
 /pony-sync-helper
 /pony-sync-helper.dSYM
 environment
+_repos/
+_corral
```

Comment:
Is there a reason this directory does not have a trailing slash?

Reply:
It works either way, mostly oversight.